### PR TITLE
Updated docs for running tests and contirbuting guidelines

### PR DESCRIPTION
**Pull request by dipeshmsft**
Fixes Issue 
#105 

Master PR 

## Description
Adds contributing guidelines
Adds building, running and debugging tests guidelines

## Customer Impact
Allows developers to understand the structure of the repo, how to build it, run tests and make contributions.

## Regression
--

## Testing
--

## Risk
--

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/wpf-test/pull/132)

### CODE REVIEW

**Review by singhashish-wpf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## Reporting Issues
+
+We always welcome bug reports and overall feedback. Here are a few tips on how you can make reporting your issue as effective as possible.
+
+Test failures can mean either of two things: 
+- There is a bug in WPF product itself.
+- Or, the tests are failing due to test infra issues ( tests may not have been updated to reflect new behaviour in WPF, test infra is failing, updates in drivers and dependencies).
+
+For any such case, please [file an issue]() in this repository. Depending on the nature of failure, the issue may be transferred to dotnet/wpf repo.
```
Comment (pchaurasia14) [inline CONTRIBUTING.md:13]: link address is missing

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+### Writing a Good Bug Report
+
+Good bug reports make it easier for maintainers to verify and root cause the underlying problem. The better a bug report, the faster the problem will be resolved. Ideally, a bug report should contain the following information:
+
+* A high-level description of the problem.
+* A _minimal reproduction_, i.e. the smallest size of code/configuration required to reproduce the wrong behavior. Here, we expect the command that is used for running the tests.
+* Information on the environment: OS/distro, CPU arch, SDK version, Screen Resolution, Multiple Screen Setup, RDP Connection etc.
+* In case of test failures, please add the /Area, /SubArea, /Name of the tests. 
+
+When ready to submit a bug report, please use the [Bug Report issue template]().
```
Comment (pchaurasia14) [inline CONTRIBUTING.md:30]: Need to fill destination address

File: README.md [hunk 3]
```diff
@@ -10,13 +10,9 @@ This repo is currently under the process of being open-sourced. We are tracking
 
 ## Getting started
 
-* [Building, Debugging and Testing](docs/getting-started.md)
-* [Test Infrastructure Notes](docs/test-infrastructure.md)
-* [Test Suites and Test Categorization](docs/tests-suites.md)
-* [New Test Addition Guideline](docs/test-addition-guidelines.md)
-
+Follow the [Developer Guide](docs/developer-guide.md) instrutions for machine setup and understanding the workflow.
```
Comment (pchaurasia14) [inline README.md:13]: spell check

File: docs/building-running-debugging.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+- `Summary.xml` : Contains the area-wise summary of the run.
+- `testResults.xml` : Contains the result of all the tests that were run. If the test failed, you can find it here. You will also get the repro arguments related to the failing test here.
+ 
+## Running tests on locally built WPF assemblies
+
+For running tests on locally built WPF assemblies, the rest of the process ( building and running tests will remain the same ), however, we will need to replace the installed WPF binaries with our locally built ones. Once done, run the tests as mentioned above.
+
+### Replace locally built WPF binaries to your local SDK installation
+
+Clone and build [WPF]() repo using `build.cmd` and copy the resulting assembly(-ies) to your local SDK installation
```
Comment (pchaurasia14) [inline docs/building-running-debugging.md:189]: missing address

**Pull request merged**